src/CRUD.js

```diff
@@ -0,0 +1,147 @@
+const newTask = document.querySelector('.task-input');
+const add = document.querySelector('.add');
+const list = document.querySelector('.lists');
+const reload = document.querySelector('.fa-arrows-rotate');
+
+const completed = false; let
+  index = 1;
+let listArr = [];
+
+const showTasks = () => {
+  const storedData = localStorage.getItem('data');
+  // index = 0;
+  if (storedData === null) {
+    listArr = [];
+  } else {
+    listArr = JSON.parse(storedData);
+    index = listArr.length + 1;
+  }
+
+  let newLiTag = '';
+  listArr.forEach((element) => {
+    if (element.completed === true) {
+      newLiTag += `<li>
+       <div class="left">
+       <input type="checkbox" id="check${element.index}" onclick="completeTask(${element.index});" checked>
+       <input type="text" class="task" id="task${element.index}" value="${element.description}" readonly>
+       </div>
+       <div class="right">
+       <i class="fa-solid fa-pen-to-square edit " id="edit${element.index}" onclick="editTask(${element.index});"></i>
+       <i class="fa-solid fa-floppy-disk save hide" id="save${element.index}" onclick="saveTask(${element.index});"></i>
+       <i class="fa-solid fa-trash delete" id="del" onclick="deleteTask(${element.index});"></i>       
+</div>
+       </li><hr>`;
+      // const checkTask = document.getElementById('task${element.index}');
+      // checkTask.style.textDecoration = 'line-through';
```

Comment:
- [ ] Kindly remove all commented-out un used/ unwanted pieces of code from your code base. It makes it look messy and cluttered.